Modules/trigger.py

```diff
@@ -83,3 +100,24 @@ def channel(self) -> str or None:
             msg (str): Message to send.
         """
         await self.bot.message(self.channel if self.channel else self.nickname, msg)
+
+    def __eq__(self, other) -> bool:
+        if self is other:
+            return True
+        elif isinstance(other, Trigger):
+            for name, value in Trigger.__dict__.items():
+                if isinstance(value, property):
+                    if getattr(self, name) != getattr(other, name):
+                        return False
+            else:
+                return True
+        else:
```

Comment:
If `other` is not of Type Trigger this should raise a TypeError rather than calling the super

Reply:
Unless there is convention which I am not aware of, there should not be a problem with this.
By definition, calling the super follows the convention of returning `False` if the other object is of a different type which Python's buitins follow.
An argument could be made that it should just return `False` without asking the super, since in the case of subclassing object, the two are equivalent.

Reply:
The two methods would be equal, but i see no reason to invoke the super in this context. would it not be more efficient to simply return false, since the super call is both unnecessary and yield the same result?

Reply:
Hmm, alrighty